.github/actions/dependabot-coverage-check/dependabot-coverage-check.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# dependabot-coverage-check
+# Author: wojciech.piotrowiak
+# Description: Workaround for https://github.com/dependabot/dependabot-core/issues/6345.
+# This script checks if all GitHub Actions in .github/actions are listed in .github/dependabot.yml.
```

Comment:
update to keep them consistent

anyway I am not so inclined to duplicate these info here given they are already provided in the action.yml